bbc/simorgh
# WSTEAM1-828: Add media loader placeholder

Resolves JIRA [WSTEAM1-828](https://jira.dev.bbc.co.uk/browse/WSTEAM1-828)

Overall changes
======
Hoists the media Placeholder component to TypeScript.

Code changes
======

- Converts media placeholder component to Typescript
- Adds additional `className` props to Legacy components to override styling applied via the `styled` emotion api
- Creates a global const for `no-js`
- Updates unit tests

Testing
======
1. Visit a live page http://localhost.bbc.com:7081/pidgin/live/c7p765ynk9qt?renderer_env=test
2. Check if any regression against our normal media player
3. Check that placeholder logic works correctly

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
